experimentalSetups/head_face-v3/camera/dragonfly2_config_left.ini

```diff
@@ -1,4 +1,5 @@
 device dragonfly2
+capabilities RAW

```

Comment:
For `dragonfly2` the parameter `capabilities` as to be set to `COLOR` or not set (`COLOR` is default)